app/src/main/res/layout/vlc_player_interface.xml

```diff
@@ -74,32 +74,55 @@
             android:orientation="horizontal" />
     </FrameLayout>
 
-    <RelativeLayout
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:id="@+id/topPanel"
         android:layout_width="fill_parent"
         android:layout_height="155dp"
         android:layout_gravity="center_horizontal|top"
+        android:layout_marginHorizontal="40dp"
+        android:layout_marginTop="20dp"
         android:clickable="false"
         android:visibility="invisible">
 
+        <ImageView
+            android:id="@+id/item_logo"
+            android:layout_width="wrap_content"
+            android:layout_height="60dp"
+            app:layout_constraintEnd_toStartOf="@id/textClock"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintHorizontal_bias="0.0" />
+
+        <TextView
+            android:id="@+id/item_title"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
```

Comment:
When the title of a tv series it long it will append a new line. I'm not 100% sure if we want to keep that behavior or keep it at a single line.

```suggestion
            android:layout_height="wrap_content"
            android:ellipsize="end"
            android:singleLine="true"
```

I guess the same question for the subtitle

Reply:
It would have to be a _very_ long title... I think wrapping would be fine for those personally :man_shrugging:

Reply:
Sounds good, I found a few things in my library with long titles but they never went over 2 lines.